jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -54,12 +54,23 @@ def functionTest(String test_name, String TEST_GROUP, Boolean RUN_FIT_TEST, Bool
             try{
                 timeout(90){
                     // run test script
-                    sh '''#!/bin/bash -x
-                    pushd build-config
-                    ./build-config
-                    popd
-                    ./build-config/test.sh
-                    '''
+                    def repos = env.REPOS_UNDER_TEST.tokenize(',')
```

Comment:
@panpan0000 Dear peter, please refer to this one
http://10.62.59.175:8080/blue/organizations/jenkins/image-service/detail/image-service/195/pipeline

The only point is that: the failure is not caused by the code change in on-build-config, but the configuration in Jenkins and that changes in RackHD.git